sphinx_js/renderers.py

```diff
@@ -240,14 +242,15 @@ def _returns_formatter(field, description):
 
 def _params_formatter(field, description):
     """Derive heads and tail from ``@param`` blocks."""
-    heads = ['param']
-    types = _or_types(field)
-    if types:
-        heads.append(types)
-    heads.append(field['name'])
+    heads = ['param', field['name']]
     tail = description
     return heads, tail
 
+def _param_type_formatter(field, description):
+    """Derive heads and tail from ``@param`` blocks."""
```

Comment:
I think you just copied and pasted the docstring from the above accidentally. :-)